astropy/coordinates/builtin_frames/ecliptic_transforms.py

```diff
@@ -37,15 +37,24 @@ def _mean_ecliptic_rotation_matrix(equinox):
 
 
 def _true_ecliptic_rotation_matrix(equinox):
-    # This code calls pnm06a from ERFA, which retrieves the precession
-    # matrix (including frame bias) according to the IAU 2006 model, and
-    # including the nutation. This family of systems is less popular
+    # This code calls the same routines as done in pnm06a from ERFA, which
+    # retrieves the precession matrix (including frame bias) according to
+    # the IAU 2006 model, and including the nutation.
+    # This family of systems is less popular
     # (see https://github.com/astropy/astropy/pull/6508).
     jd1, jd2 = get_jd12(equinox, 'tt')
-    rnpb = erfa.pnm06a(jd1, jd2)
-    _, nut_obl = erfa.nut06a(jd1, jd2)*u.radian
-    obl = erfa.obl06(jd1, jd2)*u.radian + nut_obl  # calculate the true obliquity of the ecliptic
-    return matrix_product(rotation_matrix(obl, 'x'), rnpb)
+    # Here, we call the three routines from erfa.pnm06a separately,
+    # so that we can keep the nutation for calculating the true obliquity
+    # (which is a fairly expensive operation); see gh-11000.
+    # pnm06a: Fukushima-Williams angles for frame bias and precession.
+    gamb, phib, psib, epsa = erfa.pfw06(jd1, jd2)
```

Comment:
Is there any harm in expanding these out? I find the short variable names used internally by sofa to hinder easy reading of the code, we could use `gamma_bar`, `phi_bar`, `psi_bar` and `epsilon_A` for example, which are easier to cross-ref with the Fukushima paper.

Reply:
This is indeed tricky. In the end, I think using the variable names as in ERFA is most useful, as I'd think that is the first thing people might look at. And in this case, the F-W names are not clearer in themselves, only by reference. So, maybe just a comment cross-linking them? I've done that for now.